shopware/Caddyfile

```diff
@@ -0,0 +1,45 @@
+DOMAIN {
+    gzip {
+        level 6
+    }
+    expires {
+        match .css$ 1m
+        match .js$ 1m
+        match .png$ 1m
+        match .jpg$ 1m
+        match .gif$ 1m
+        match .ico$ 1m
+        match .woff$ 1m
+        match .woff2$ 1m
+        match .ttf$ 1m
+        match .eof$ 1m
+        match .pdf$ 1s
+        match .txt$ 1s
+        match .html$ 1s
+    }
```

Comment:
Of course not, feel free to reduce it. Sorry :+1: 
Internal and rewrites are, but status only for some kind of additional security.